mlfinlab/structural_breaks/sadf.py

```diff
@@ -10,13 +10,15 @@
 
 # pylint: disable=invalid-name
 
-def _get_sadf_at_t(X: pd.DataFrame, y: pd.DataFrame, min_length: int) -> float:
+def _get_sadf_at_t(X: pd.DataFrame, y: pd.DataFrame, min_length: int, model: str, phi: float) -> float:
     """
     Snippet 17.2, page 258. SADF's Inner Loop (get sadf value at t)
```

Comment:
Since you have added an extension to the code base and because the docstrings here are incomplete. I invite you to write the docstring for this function so that the end user can better understand what it does.